libs/tx-builder-feature/src/utils/txBuilderUtils.ts

```diff
@@ -199,6 +212,7 @@ export async function handleFireTx({
     provider.getSigner().connectUnchecked()
   );
 
-  const ethersTx = await contract.functions[txName](...args);
-  return executeTx(tx, ethersTx, setTransactions, chainId);
+  const ethersTx = await contract.functions[method](...realArgs);
```

Comment:
Not needed right now, But I think this is likely were we want to send the gasLimit when we get to processProposal down the line.

Can use the overrides object:
https://docs.ethers.io/v5/api/contract/contract/#contract-functionsSend

Reply:
Thinking the same. Am I correct in assuming that this is only crucial for when we call processProposal?

Reply:
> Thinking the same. Am I correct in assuming that this is only crucial for when we call processProposal?

Yeah that is the only one we'll need it on.